src/foam/nanos/notification/services

```diff
@@ -113,3 +113,51 @@ p({
   "serviceClass":"foam.nanos.notification.email.POP3EmailService",
   "client":"{ \"class\":\"foam.nanos.notification.email.ClientPOP3EmailService\", \"delegate\": { \"class\":\"foam.box.HTTPBox\", \"url\":\"service/pop3\" } }"
 })
+
+p({
+  "class":"foam.nanos.boot.NSpec",
+  "name":"localSmsMessageDAO",
+  "serviceScript":"""
+    return new foam.dao.EasyDAO.Builder(x)
+      .setOf(foam.nanos.notification.sms.SMSMessage.getOwnClassInfo())
+      .setAuthorize(false)
+      .setSeqNo(true)
+      .setJournalType(foam.dao.JournalType.SINGLE_JOURNAL)
+      .setJournalName(\"smsMessages\")
+      .build();
+  """
+})
+
+p({
+  "class":"foam.nanos.boot.NSpec",
+  "name":"smsMessageDAO",
+  "serve":true,
+  "serviceScript":"""
+    return new foam.dao.EasyDAO.Builder(x)
+      .setOf(foam.nanos.notification.sms.SMSMessage.getOwnClassInfo())
+      .setInnerDAO((foam.dao.DAO) x.get(\"localSmsMessageDAO\"))
```

Comment:
all the \" are not required when the script is inside the """

Reply:
you mean backslashes aren't required, right?

Reply:
yes.